ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -466,9 +470,7 @@ void NGraphEncapsulateOp::ComputeUsingParallelExecutor(OpKernelContext* ctx) {
       // Get the set of IO tensors for the next iteration
       std::tuple<int, PipelinedTensorVector, PipelinedTensorVector>
           io_tensors_next_iter;
-      OP_REQUIRES_OK(ctx, m_parallel_executor->GetTensorsFromPipeline(
-                              ng_exec, io_tensors_next_iter));
-
+      io_tensors_next_iter = pipelined_tensor_store->get_tensors();
```

Comment:
there is a bunch of get<0>, get<1> etc in the code.
Lets use `struct IoTensorBundle` or std::tie to extract the elements of the tuple (or maybe even change the code of `pipelined_tensor_store->get_tensors` to return `IoTensorBundle`, but that may be beyond the scope of this PR)